services/js/src/main/scala/org/apache/gearpump/dashboard/DashboardApp.scala

```diff
@@ -0,0 +1,54 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.gearpump.dashboard
+
+import com.greencatsoft.angularjs.Angular
+import org.apache.gearpump.dashboard.services._
+
+import scala.scalajs.js.JSApp
+import scala.scalajs.js.annotation.JSExport
+
+@JSExport
+object DashboardApp extends JSApp {
```

Comment:
Agreed, there is some risk on SNAPSHOT's. What issues are relevant to the discussion? Is the discussion whether to use scalajs-angular at all or whether xmlhttprequest should be always used regardless of whether we're using angular, reactive, etc? Changing to XmlHttpRequest and then changing back to $http is not a very efficient use of time if the discussion can be resolved without doing this. Perhaps we need to ask the author of scalajs-angular when he plans on 0.5 or are there other concerns with using HttpService or $http?